src/parser/demonhunter/havoc/modules/talents/GlaiveTempest.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import TalentStatisticBox from 'interface/others/TalentStatisticBox';
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import STATISTIC_ORDER from 'interface/others/STATISTIC_ORDER';
+import { formatThousands } from 'common/format';
+import Events from 'parser/core/Events';
+
+class GlaiveTempest extends Analyzer {
+
+    damage = 0;
+
+    constructor(...args) {
+        super(...args);
+        this.active = this.selectedCombatant.hasTalent(SPELLS.GLAIVE_TEMPEST_TALENT.id);
+        if (!this.active) {
+          return;
+        }
+        this.addEventListener(Events.damage.by(SELECTED_PLAYER).spell(SPELLS.GLAIVE_TEMPEST_DAMAGE), this.onDamageEvent);
+      }
+
+    onDamageEvent(event) {
+        this.damage += event.amount;
+      }
+
+    statistic() {
```

Comment:
TalentStatisticBox is deprecated. You should be able to find examples of the new Statistic boxes in any of the mage talent modules.

Essentially you use Statistic instead and set the category to STATISTIC_CATEGORIES.TALENTS

Reply:
Thanks for the feedback, updated accordingly.